src/api/app/views/webui/main/_status_messages.html.haml

```diff
@@ -7,7 +7,7 @@
         %i.fa.fa-rss
   .list-group
     = render partial: 'status_message', collection: status_messages
-    - if policy(StatusMessage.new).create? && !feature_enabled?(:responsive_ux)
+    - if policy(StatusMessage.new).create? # && !feature_enabled?(:responsive_ux)
```

Comment:
Everything after `#` should be removed, right?

Reply:
Oh yeah, i forgot that :facepalm: